packages/inference/src/providers/replicate.ts

```diff
@@ -21,7 +21,7 @@ export const REPLICATE_SUPPORTED_MODEL_IDS: ProviderMapping<ReplicateId> = {
 			"stability-ai/sdxl:7762fd07cf82c948538e41f63f77d685e02b063e37e496e96eefd46c929f9bdc",
 	},
 	"text-to-speech": {
-		"OuteAI/OuteTTS-0.3-500M": "jbilcke/oute-tts:39a59319327b27327fa3095149c5a746e7f2aee18c75055c3368237a6503cd26",
+		"hexgrad/Kokoro-82M": "jaaari/kokoro-82m:dfdf537ba482b029e0a761699e6f55e9162cfd159270bfe0e44857caa5f275a6",
```

Comment:
```suggestion
		"hexgrad/Kokoro-82M": "jaaari/kokoro-82m:dfdf537ba482b029e0a761699e6f55e9162cfd159270bfe0e44857caa5f275a6",
		"OuteAI/OuteTTS-0.3-500M": "jbilcke/oute-tts: 3c645149db020c85d080e2f8cfe482a0e68189a922cde964fa9e80fb179191f3",
```

actualy @jbilcke-hf just updated the param name, so adding it back